### DISCUSSION TITLE
为你的 Astro 博客添加评论功能 - https://liruifengv.com/posts/add-comments-to-astro/

### DISCUSSION THREAD

**Discussion by giscus** [liruifengv/liruifengv.com#27] [Announcements]
# 为你的 Astro 博客添加评论功能 - https://liruifengv.com/posts/add-comments-to-astro/

使用 giscus 为你的 Astro 博客添加评论功能。

https://liruifengv.com/posts/add-comments-to-astro/

**Comment by Zhengqbbb** [comment]
1. DOMContentLoaded 注入 srcipt
2. 监听主题

https://github.com/Zhengqbbb/qbb.sh/commit/cba83020b004e46fb4d7ded26ba80ca51e3d1fa3

**Comment by facl** [comment]
谢谢，已按照教程搭建
